convert_to_our_format.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+aliasesFields = ['preferred synonym', 'synonyms', 'preferred fully specified name', 'fully specified names']
+f = open('snomed/concepts.json')
+concept_dict = json.load(f)
+f = open('snomed/is_a.json')
+is_a_dict = json.load(f)
+f = open('snomed/concept_without_father.json')
+concept_without_father = json.load(f)
+id = 1
+
+
+def create_hierarchy(conc_id):
+    global id
+    id = id + 1
+    if conc_id not in is_a_dict:
+        aliases = {}
+        for alias in aliasesFields:
+            if len(concept_dict[conc_id][alias]) > 0:
+                aliases[alias] = concept_dict[conc_id][alias]
+        concept_leaf = {"name": concept_dict[conc_id]["name"], "id": id, "state": {"aliases": aliases}, "children": []}
+        return concept_leaf
+
+    aliases = {}
+    for alias in aliasesFields:
+        if len(concept_dict[conc_id][alias]) > 0:
+            aliases[alias] = concept_dict[conc_id][alias]
+    concept = {"name": concept_dict[conc_id]["name"], "id": id, "state": {"aliases": aliases}, "children": []}
+
+    children = is_a_dict[conc_id]
+    for c in children:
+        concept["children"].append(create_hierarchy(c))
+
+    return concept
+
+
+def create_our_format():
+
+    root = {"name": "root", "id": 1, "state": {"aliases": {aliasesFields[0]: ["all"]}}, "children": []}
+
+    if len(concept_without_father) == 1:
```

Comment:
remember we said this code needs to be generic. maybe in other taxonomies we would want to keep their top single root to be under our root?
instead we need to behave here equally to any format = simply do what you did in the `else` block of code - and remove the entire `if` part. then in the snomed_extract code you should this logic where you remove the head and return its immediate children as the list of heads
the place you know you want to do it - thats where it should happen :)